akka-http-core/src/main/scala/akka/http/impl/model/parser/CharacterClasses.scala

```diff
@@ -49,6 +49,7 @@ private[http] object CharacterClasses {
   val `relaxed-fragment-char` = VCHAR -- '%'
   val `relaxed-path-segment-char` = VCHAR -- "%/?#"
   val `relaxed-query-char` = VCHAR -- "%&=#"
+  val `relaxed-query-value-char` = VCHAR -- "%&#"
```

Comment:
To keep it consistent with the [rename below](https://github.com/akka/akka-http/pull/1190/files#diff-52e496fada3f67a2626c662c33c0d392R87) rename `relaxed-query-char` to `relaxed-query-key-char` and do the same for `strict-query-char`.